spec/features/admin/tag_rules_spec.rb

```diff
@@ -189,25 +195,25 @@
       expect(default_fsm_tag_rule.preferred_matched_shipping_methods_visibility).to eq "hidden"
 
       # FilterShippingMethods rule
-      expect(fsm_tag_rule.reload.priority).to eq 1
+      expect(fsm_tag_rule.reload.priority).to eq 4
```

Comment:
Thanks for telling me about these. 
Next time I will confirm with you first before I change any unfamiliar failed tests.